test/fixtures/models/katello_pools.yml

```diff
@@ -2,13 +2,13 @@ pool_one:
   cp_id: "abc123"
   created_at: <%= Time.now %>
   updated_at: <%= Time.now %>
-  subscription_id: <%= ActiveRecord::FixtureSet.identify(:basic_subscription) %> 
```

Comment:
None of the changes in this file are needed :)

Reply:
oh no. That was unintentional. Thank you!